File: Python/tests/test_prl_rp_multipleB.py
import pytest

from hbayesdm.models import prl_rp_multipleB


def test_prl_rp_multipleB():
    _ = prl_rp_multipleB(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_prl_rp_multipleB.R
context("Test prl_rp_multipleB")
library(hBayesDM)

test_that("Test prl_rp_multipleB", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(prl_rp_multipleB(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
